src/args.ts

```diff
@@ -53,13 +53,19 @@ export let args : ArgDescriptor[] = [
     name: 'open',
     alias: 'o',
     description: 'The page to open in the default browser on startup.',
+    type: Boolean,
+  },
+  {
+    name: 'open-browser',
```

Comment:
as opposed to just `browser` & `path`? I always prefer shorter if possible but I value the clarity (that these are open-related options and do nothing on their own) here a bit more. Happy to change them if you strongly disagree.